workshops/models.py

```diff
@@ -1085,10 +1085,15 @@ def __str__(self):
 class Role(models.Model):
     '''Enumerate roles in workshops.'''
 
-    name       = models.CharField(max_length=STR_MED)
+    name = models.CharField(max_length=STR_MED)
+    verbose_name = models.CharField(max_length=STR_LONG,
+                                    null=False, blank=True, default='')
+    show_in_training_request_form = models.BooleanField(
+        default=False,
+        verbose_name='include in training request form?')
```

Comment:
No worries, I was only asking to understand the changes better.

Question: we use a different approach to mark badges that are for instructors. We use `BadgeQuerySet.instructor_badges` -- why don't you want to hardcode roles to be included in the form this way?